frb_dart/lib/src/helpers.dart

```diff
@@ -148,3 +148,15 @@ Duration wire2apiDuration(int ts) {
   }
   return Duration(microseconds: ts);
 }
+
+Uint8List wire2apiConcatenateBytes(List raw) {
```

Comment:
nit: `List<UuidValue>`, we can widen this later if needed

Reply:
ah so the reason I did it this way was to avoid forcing [package uuid](https://pub.dev/packages/uuid) as dependency on `frb dart`, are you sure ?

Reply:
Dart dependencies are relatively cheap to add and can be tree-shaken, so I think it should be fine.

Reply:
ok ! 👌

Reply:
ok done in de8eaac806974e1d1fc70e6a1f690c5fc9b0f01e, sorry I forgot the co-authoring thing I'm getting tired lol 😴